kubespawner/spawner.py

```diff
@@ -1895,7 +1896,6 @@ def stop(self, now=False):
         except TimeoutError:
             self.log.error("Pod %s did not disappear, restarting pod reflector", self.pod_name)
             self._start_watching_pods(replace=True)
-            raise
```

Comment:
I'm not sure if it makes sense to raise an error or not in this case. Worth knowing is that in [this case](https://github.com/jupyterhub/jupyterhub/blob/0317fd63fa1dac4d709b4f95738e74c5c408da6c/jupyterhub/spawner.py#L1572-L1575) only a warning is logged.

Reply:
I think it makes sense to keep this as well. I think doing so results in a clear indication of an issue in juppyterhub logs etc and that may be the right call. I do wonder what the outcome is for jupyterhub, it wont crash and restart right?